xapian-core/tests/harness/backendmanager.cc

```diff
@@ -42,6 +42,8 @@
 #include "index_utils.h"
 #include "backendmanager.h"
 #include "unixcmds.h"
+#include "common/stringutils.h"
+
 
 using namespace std;
```

Comment:
`common` is in the include path, so that should just be `#include "stringutils.h"`.

Also, you've added an extra blank line after it.

Reply:
fixed.